middleware.ts

```diff
@@ -17,54 +12,10 @@ export async function middleware(request: NextRequest) {
    login to the original page they were trying to get to.*/
 
   if (protectedPaths.some(path => pathname.startsWith(path))) {
-    const token = await getAuthTokenServer();
 
-    if (!token) {
```

Comment:
Removed most of the middleware.ts code until we get a better send of what we will be using it for.